### PR TITLE
Skip dpnp tests on module level

### PR DESCRIPTION
**Pull request by 1e-to**

### CODE REVIEW

File: numba_dppy/tests/integration/__init__.py [hunk 1]
```diff
@@ -16,4 +16,12 @@
...
 
+import pytest
+
+from numba_dppy.tests._helper import ensure_dpnp
+
 from . import *
+
+pytestmark = pytest.mark.xfail(not ensure_dpnp(), reason="No dpnp available")
+if ensure_dpnp():
+    import dpnp
```
Comment (PokhodenkoSA) [inline numba_dppy/tests/integration/__init__.py:27]: Use this approach:
https://github.com/IntelPython/numba-dppy/blob/19eb6fe1f402a0de85f7e26986a71c313a1a67cd/numba_dppy/tests/test_debug_dppy_numba.py#L27

Comment (1e-to) [reply numba_dppy/tests/integration/__init__.py:27]: If we use this, there is no information about which tests and number was skipped:
collected 0 items / 1 skipped

File: numba_dppy/tests/njit_tests/dpnp/__init__.py [hunk 2]
```diff
@@ -16,4 +16,12 @@
 # limitations under the License.
 ################################################################################
 
+import pytest
+
+from numba_dppy.tests._helper import ensure_dpnp
+
 from . import *
+
```
Comment (1e-to) [inline numba_dppy/tests/njit_tests/dpnp/__init__.py:24]: Duplicate with  **numba_dppy/tests/integration/__init__.py**

File: numba_dppy/tests/passes/test_rename_numpy_function_pass.py [hunk 3]
```diff
@@ -94,7 +94,8 @@ def got(a):
 
 
 @pytest.mark.skipif(
-    not ensure_dpnp(), reason="test only when dpnp is available"
+    not ensure_dpnp(),
```
Comment (1e-to) [inline numba_dppy/tests/passes/test_rename_numpy_function_pass.py:97]: Is it okay that there is exist check **ensure_dpnp**?

Comment (PokhodenkoSA) [reply numba_dppy/tests/passes/test_rename_numpy_function_pass.py:97]: You can create `skip_no_dpnp` mark in helper and reuse it.

**Comment by PokhodenkoSA** [comment]
Mostly fixed by #668. But it contains some improvements.

**Comment by PokhodenkoSA** [comment]
Closed by #675.

**Pull request closed without merging**